monetio/models/_cesm_fv_mm.py

```diff
@@ -52,6 +58,31 @@ def open_mfdataset(
     #############################
     # Process the loaded data
     # extract variables of choice
+    # If vertical information is required, add it.
+    # Assume that height and geopotential height are equal
```

Comment:
```suggestion
```
Not sure if this comment is needed, since the calculation is correct for geopotential height (which Z3 is in), and the fact that it is *geopotential* height is in the long name.